src/js/components/Chart/Chart.js

```diff
@@ -218,6 +217,9 @@ const Chart = React.forwardRef(
       ];
     };
 
+    const useGradient = color && Array.isArray(color);
+    const patternId = pattern && `${pattern}-${id}-pattern`;
```

Comment:
Two questions:
1. Should we document that `id` is required when using `pattern`?
2. Shouldn't `patternId` be relevant only when `type` is 'area' ? why is it calculated for all the other use cases?

Reply:
yes to both, fixed